deepmd/pt/loss/ener.py

```diff
@@ -411,3 +411,75 @@ def label_requirement(self) -> list[DataRequirementItem]:
                 )
             )
         return label_requirement
+
+
+class EnergyHessianStdLoss(EnergyStdLoss):
```

Comment:
I am wondering why we cannot use the same loss class for `EnergyStdLoss` and `EnergyHessianStdLoss`, considering hession is derived from energy. @wanghan-iapcm what do you think?